k2pdfoptlib/bmpregion.c

```diff
@@ -640,6 +683,47 @@ exit(10);
     */
     }
 
+/*
+** v2.52
+** For each column, only count the top-most and bottom-most black pixels in that column
+** into the rowcount[] stats.  This prevents lowercase height from getting fooled by
+** letters like e, a, s, A, H.
+**
+** DECIDED THIS DIDN'T REALLY HELP

```

Comment:
Out of curiosity, who decided this?

Reply:
Let me check… (TODO: update the `update_upstream.sh*` script to use a dedicated `k2pdfopt` commit identity).

Reply:
This come from upstream.

Reply:
Right, thanks.

Reply:
From 2.52 to be precise.